cordova-js-src/splashscreen.js

```diff
@@ -69,18 +69,21 @@ function readPreferencesFromCfg(cfg, manifest) {
         splashImageSrc = schema + ':///' + manifest.getSplashScreenImagePath().replace(/\\/g, '/');
 
         bgColor = cfg.getPreferenceValue('SplashScreenBackgroundColor') || bgColor;
-        bgColor = bgColor.replace('0x', '#').replace('0X', '#');
-        if (bgColor.length > 7) {
-            // Remove aplha
-            bgColor = bgColor.slice(0, 1) + bgColor.slice(3, bgColor.length);
-        }
+        bgColor = bgColor.toLowerCase().replace('0x', '#');
```

Comment:
This unrelated change doesn't add the possibility of other bugs?

Reply:
I had to force the case of bgColor to compare it to 'transparent' so I combined it with the replacement of '0x' and '0X'. bgColor is used as CSS color value which I believe is case-insensitive.